test/app/dftb+/tests

```diff
@@ -128,7 +128,8 @@ hybrid/periodic/Acene-gamma-neigh-force-cam           #? not WITH_MPI
 hybrid/periodic/Acene-gamma-neigh-force-hyb           #? not WITH_MPI
 hybrid/periodic/Acene-gamma-neigh-force-lc            #? not WITH_MPI
 hybrid/periodic/Acene-kpts-matrix-lc                  #? MPI_PROCS <= 1
-hybrid/periodic/Acene-kpts-matrix-restart             #? MPI_PROCS <= 1
+hybrid/periodic/Acene-kpts-matrix-restart-I           #? MPI_PROCS <= 1
+hybrid/periodic/Acene-kpts-matrix-restart-II          #? MPI_PROCS <= 1
```

Comment:
Might be useful to have a comment somewhere why there are two separate versions of this test (either here, in the git log or ???) or rename by mixer type instead of I and II.